site/docs/about/enterprise-release-notes.md

```
@@ -1,8 +1,20 @@
----
-description: "Release notes for Lakekeeper+, the commercial distribution, covering Cedar authorization, UI branding, admission gates and other enterprise features."
----
+# Lakekeeper Plus Release Notes
 
-# Lakekeeper+ Release Notes
+## Unreleased
+
+### Features
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)